akka-stream-tests/src/test/scala/akka/stream/scaladsl/FlowFlattenMergeSpec.scala

```diff
@@ -0,0 +1,129 @@
+/**
+ * Copyright (C) 2015 Typesafe Inc. <http://www.typesafe.com>
+ */
+package akka.stream.scaladsl
+
+import akka.stream.testkit.AkkaSpec
+import akka.stream.ActorMaterializer
+import scala.concurrent._
+import scala.concurrent.duration._
+import org.scalatest.concurrent.ScalaFutures
+import org.scalactic.ConversionCheckedTripleEquals
+import akka.stream.testkit.TestPublisher
+import org.scalatest.exceptions.TestFailedException
+import akka.stream.testkit.scaladsl.TestSink
+
+class FlowFlattenMergeSpec extends AkkaSpec with ScalaFutures with ConversionCheckedTripleEquals {
+  implicit val mat = ActorMaterializer()
+  import system.dispatcher
+
+  def src(i: Int) = Source((10 * i) until (10 * i + 10))
```

Comment:
i would remove the multiplication. `Source(List(src(0), src(10), src(20), src(30)))` is easier for me to mentally relate to the actual ranges of 10.